e2e/app/component/tiered-menu.ts

```diff
@@ -1,12 +1,42 @@
 import Container from 'app/container';
-import {Page} from 'puppeteer';
+import {ElementHandle, Page} from 'puppeteer';
 
-const defaultXpath = '//*[contains(concat(" ", normalize-space(@class), " "), " p-tieredmenu ")]';
+const defaultXpath = '//*[contains(concat(" ", normalize-space(@class)), " p-tieredmenu") ' +
+   'and contains(concat(" ", normalize-space(@class)), " p-menu-overlay-visible")]';
 
 export default class TieredMenu extends Container {
 
   constructor(page: Page, xpath: string = defaultXpath) {
     super(page, xpath);
   }
 
+  /**
+   * Select menu item(s).
+   * @param {ElementHandle} menuParentElement
+   * @param {string | string[]} selectMenuItems
+   */
+  async select(selectMenuItems: string[]): Promise<void> {
```

Comment:
It looks like what this function does is to find and click all items in a list of items - can you change the name of the function to indicate that that is what it does?

Reply:
user probably doesn't care about there is an internal helper function for finding menuitems. do you recommend a better name? I'd have to rename other `select()` functions too to keep naming convention standard.
https://github.com/all-of-us/workbench/blob/4091a0588aabe68831499fe337a16c33a2ce4e06/e2e/app/component/select-menu.ts#L26

Reply:
Developers are the users here and as a developer I'd prefer to have meaningful function names. I would call this `clickMenuItems`, and for other types of inputs I would similar functions things like `clickRadioButton` or `clickCheckBox`.

Reply:
Sounds good.
I renamed following three function names (for consistency):
- `selectAction` to `clickAction` in `EllipsisMenu.ts`.
- `select` to `clickMenuItem` in `SelectMenu.ts`.
- `select` to `clickMenuItem` in `TieredMenu`.